markdown/events/2022/hackathon-march-2022.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Hackathon - March 2022
+subtitle: A virtual online hackathon to develop nf-core together
+type: hackathon
+start_date: "2022-03-16"
+start_time: "10:00 CET"
+end_date: "2022-03-18"
+end_time: "18:00 CET"
+location_name: Gather town and Slack.
+---
+
+
+Please join us for the March 2022 nf-core hackathon!
+This event is entirely virtual (we will be back in gather.town!) and we are attempting to support people in all timezones.
+:earth_americas: :earth_asia: :earth_africa:
+
+More details will come soon, registration will open February 2022.
+Please check back here for details and keep an eye out on twitter and on Slack,
+but in the mean time - book the dates in your calendar!
+
+# Schedule
+
+We expect people to come and go during the hackathon due to diverse time zones.
+Please just do whatever works best for you!
+
+All locations described in the table below refer to places in the Hackathon gather.town
+space.
+
+The following schedule should display times in your local time zone:
+
```

Comment:
```suggestion
The following schedule should display times in your local time zone:

> **NB: This is a draft schedule and subject to change**

```

Reply:
I half wonder if we just comment out the entire schedule until @ggabernet has looked over it. But maybe it's fine. Happy to merge and we can always take it out again.